Community Page (#56)
scitt-community/scitt-web: community.md

---
layout: home
title: Community
permalink: /community
---

# About

## The SCITT Community

> The SCITT Community, the people behind this website and its efforts, are volunteers passionate about not only [the official SCITT standards](#the-ietf-scitt-working-group), but a variety of supply chain and transparency technologies.

Reviewer: At first, I thought the link was broken, then realized it was a heading link just below. Suggesting the following, including making external links popup in a new window.
```suggestion
The SCITT Community, the people behind this website and its efforts, are volunteers passionate about not only the [IETF SCITT standards](https://datatracker.ietf.org/doc/draft-ietf-scitt-architecture/){:target="_blank"} but a variety of supply chain and transparency technologies.
```

Author: I mean, I was trying for the former, but it does not make that much of a difference to me. I will integrate your recommended change.

Author: I also generally dislike forced opening in a new tab, but I am fine with this change for that behavior with `target=_blank` for various links, but is this something not better added in the underlying Jekyll theme in the future?

Reviewer: For external links, I was using popups for consistency, while internal links were simple navigation. I was looking for a generic jekyll way to have all internals be nav, while all externals be new-tabs. But the closest I found was the target=blank trick.

## The IETF SCITT Working Group

> The IETF Supply Chain Integrity, Transparency, and Trust Working Group is the group of volunteers who formally maintain the [SCITT Architecture](https://datatracker.ietf.org/doc/draft-ietf-scitt-architecture/), [the SCITT Reference API](https://datatracker.ietf.org/doc/draft-ietf-scitt-scrapi/), and other draft specifications and finalized standards for the scope of work defined in [its current charter](https://datatracker.ietf.org/wg/scitt/about/).

Reviewer: ```suggestion
The IETF Supply Chain Integrity, Transparency, and Trust Working Group is the group of volunteers who formally maintain the [SCITT Architecture](https://datatracker.ietf.org/doc/draft-ietf-scitt-architecture/){:target="_blank"}, the [SCITT Reference API](https://datatracker.ietf.org/doc/draft-ietf-scitt-scrapi/){:target="_blank"}, and other draft specifications and finalized standards for the scope of work defined in [its current charter](https://datatracker.ietf.org/wg/scitt/about/){:target="_blank"}.
```

# Getting Involved 

## The SCITT Community

> There are multiple ways to involve yourself in our community.
> 
> - Create issues, request code changes on pull requests, or comment on them [in our GitHub repos](https://github.com/scitt-community/).
> - Join [our groups.io mailing list](https://groups.io/g/scitt-community), give feedback on the email threads of others, or start your very own.
> - Attend our bi-weekly community meetings to discuss a variety of topics, ask questions, and better understand where your interests intersect with the group. We [meet online](https://meet.google.com/rek-mbak-nxv) every two weeks at [16:00 UTC](https://time.is/compare/300PM_5_June_2024_in_UTC/Tokyo/California/New_York/London) and more information can be found in our [online iCalendar feed](/assets/schedule.ics).

Reviewer: ```suggestion
There are multiple ways to involve yourself in our community:

- Create issues, request code changes on pull requests, or comment on them [in our GitHub repos](https://github.com/scitt-community/){:target="_blank"}.
- Join [our groups.io mailing list](https://groups.io/g/scitt-community){:target="_blank"}, give feedback on the email threads of others, or start your very own.
- Attend our bi-weekly community meetings to discuss a variety of topics, ask questions, and better understand where your interests intersect with the group. We [meet online](https://meet.google.com/rek-mbak-nxv){:target="_blank"} every two weeks at [16:00 UTC](https://time.is/compare/300PM_5_June_2024_in_UTC/Tokyo/California/New_York/London){:target="_blank"} and more information can be found in our [online iCalendar feed](/assets/schedule.ics).
```

## The IETF SCITT Working Group

> If you want to provide feedback or directly contribute to the core SCITT specifications, IETF has [general guidance](https://www.ietf.org/participate/) for how to participate and [the SCITT work group has a dedicated page](https://datatracker.ietf.org/wg/scitt/about/) about its charter, specifications, mailing lists, Zulip chat, and other ways to engage.

Reviewer: ```suggestion
If you want to provide feedback or directly contribute to the core SCITT specifications, IETF has [general guidance](https://www.ietf.org/participate/){:target="_blank"} for how to participate and [the SCITT work group has a dedicated page](https://datatracker.ietf.org/wg/scitt/about/){:target="_blank"} about its charter, specifications, mailing lists, Zulip chat, and other ways to engage.
```
